src/packages/@ncigdc/dux/auth.js

```diff
@@ -15,8 +15,16 @@ const TOKEN_REQUEST = 'gdc/TOKEN_REQUEST';
 const TOKEN_SUCCESS = 'gdc/TOKEN_SUCCESS';
 const TOKEN_FAILURE = 'gdc/TOKEN_FAILURE';
 const TOKEN_CLEAR = 'gdc/TOKEN_CLEAR';
+const isDev = process.env.NODE_ENV === 'development';
```

Comment:
use this here https://github.com/NCI-GDC/portal-ui/pull/2331/files#diff-eae22e1881beee47dc829cd9b6228a6eR242